dags/veda_data_pipeline/groups/processing_tasks.py

```diff
@@ -36,5 +36,9 @@ def submit_to_stac_ingestor_task(built_stac: dict):
     return event
 
 
-
-
+@task(max_active_tis_per_dag=5)
+def build_stac_task(payload):
+    from veda_data_pipeline.utils.build_stac.handler import stac_handler
```

Comment:
Any reason to not import this at the top of the file? 😯

Reply:
When the scheduler reads the DAGs and the tasks to be scheduled, it executes the imports if they are at the top of the script. To keep the scheduler lightweight, we avoid installing non-Airflow libraries on it. Instead, we import the required libraries at the task level, where the worker executes the function. This approach is explained [here](https://github.com/NASA-IMPACT/veda-data-airflow/blob/dev/dags/veda_data_pipeline/utils/s3_discovery.py#L71C16-L71C30).